sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -2,9 +2,6 @@
 
 ## 12.12.0b1 (Unreleased)
 
-### Features Added
```

Comment:
Nit (but also I think ci might fail for this?): Can you leave the `Features Added` section and just delete this line? This will help with merging into feature branch as well as I think ci may fail if this isn't present?

Applies for other CHANGELOGS as well (expect Queue)

Reply:
Can do! I didn't know the procedure of whether to leave it or not, so I guessed, and as it turns out, the coinflip wasn't in my favor :)